### PR TITLE
Remove direct dependency on xdr

### PR DESCRIPTION
**Pull request by wpears**
Now that all browsers we support for JS have fetch, we don't need to pull in fetch-alikes.

This doesn't truly remove xdr (unfortunately) as it's still required upstream by chart-builder, but is a step towards it.

### CODE REVIEW

**Comment by anselmbradford** [comment]
What do I need to do to test this, or are checks passing enough?

**Comment by wpears** [comment]
Checks and tests should be enough

File: cfgov/unprocessed/js/organisms/MortgagePerformanceTrends/utils.js [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-let counties;
-let metros;
-let nonMetros;
-let states;
 let globalZoomLevel = 10;
 
+/**
+ * @param {string} dataType - dataType to append to the BASE_URL
+ * @param {Function} cb - The callback to pass the data
+ * @returns {undefined} The result of calling the callback on the resolved data
```
Comment (anselmbradford) [inline cfgov/unprocessed/js/organisms/MortgagePerformanceTrends/utils.js:9]: Trivial: Returns a `{Promise}`, no?

Comment (anselmbradford) [reply cfgov/unprocessed/js/organisms/MortgagePerformanceTrends/utils.js:9]: Oh I see it has two return types. So I think `{*|Promise}` or `{*}` would be more correct.

Comment (wpears) [reply cfgov/unprocessed/js/organisms/MortgagePerformanceTrends/utils.js]: I just changed it to always return a promise in a90ecdc :)

Comment (anselmbradford) [reply cfgov/unprocessed/js/organisms/MortgagePerformanceTrends/utils.js]: luv it

**Review by anselmbradford — APPROVED** [review]

**Pull request merged**